devices/tuya.js

```diff
@@ -3774,4 +3774,29 @@ module.exports = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_m9skfctm']),
+        model: 'TS0601_smoke_2',
+        vendor: 'TuYa',
+        description: 'Photoelectric Smoke Detector',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        onEvent: tuya.onEventSetTime,
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.smoke(),
+            exposes.numeric('smoke_level', ea.STATE).withUnit('ppm').withDescription('Parts per million of smoke detected'),
```

Comment:
Is `smoke_concentration` a better name?

Reply:
I just had a look on the Tuya IOT page and they call it 'Detected Smoke', and I get it but that sounds like a bool value to me. I'll change it to concentration, yes. Also in doing so, I noticed on the Tuya implementation they are working the value /10 so I'll add that too.